lib/connectors/memory.js

```diff
@@ -252,6 +252,18 @@ Memory.prototype.save = function save(model, data, options, callback) {
   });
 };
 
+Memory.prototype.replace = function save(model, data, options, callback) {
```

Comment:
Oh I see, you use it from `replaceAttributes`. Hmm, I find this little bit confusing. Could you perhaps inline this inside `replaceAttributes`?